### PR TITLE
Adding recipe for django-rdkit

### PR DESCRIPTION
**Pull request by roukydesbois**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/django-rdkit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdkit/django-rdkit/archive/v{{ version }}.tar.gz
+  sha256: 9b1f13b8777fdb5262af6f8bf1b693d6362604a60514b91643b6ff82e93cc551
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<30]
```
Comment (beckermr) [inline recipes/django-rdkit/meta.yaml:15]: You cannot have this line. Use a version constraint directly on the python requirement below.

Comment (roukydesbois) [reply recipes/django-rdkit/meta.yaml:15]: Thanks!
I removed the skip line, but I didn't get where my requirement should go though, could you please explain? I didn't find it in the doc or in the example recipe.

File: recipes/django-rdkit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (beckermr) [inline recipes/django-rdkit/meta.yaml:20]:
```suggestion
    - python >=3
```

File: recipes/django-rdkit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - pip
+  run:
+    - python
```
Comment (beckermr) [inline recipes/django-rdkit/meta.yaml:23]:
```suggestion
    - python >=3
```

Comment (beckermr) [reply recipes/django-rdkit/meta.yaml]: you need both hist and run.

File: recipes/django-rdkit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdkit/django-rdkit/archive/v{{ version }}.tar.gz
+  sha256: 9b1f13b8777fdb5262af6f8bf1b693d6362604a60514b91643b6ff82e93cc551
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<30]
```
Comment (beckermr) [inline recipes/django-rdkit/meta.yaml:15]:
```suggestion
```

File: recipes/django-rdkit/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdkit/django-rdkit/archive/v{{ version }}.tar.gz
+  sha256: 9b1f13b8777fdb5262af6f8bf1b693d6362604a60514b91643b6ff82e93cc551
+
+build:
+  noarch: python
+  number: 0
```
Comment (beckermr) [inline recipes/django-rdkit/meta.yaml:14]:
```suggestion
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv"
```

**Pull request merged**